code/src/pages/products/ProductsPage.scss

```diff
@@ -0,0 +1,47 @@
+@import "@design-system";
+
+.spa-products-page {
+  padding: spacing(10);
+  background-color: var(--spa-foreground-background-color, $white);
+  display: flex;
+  flex-direction: column;
+  gap: spacing(5);
+
+  &__header {
+    font-weight: var(--spa-header-fw, $fw-semi-bold);

```

Comment:
We can use variant prop in AppTypography. It are redundant styles

Reply:
> We can use variant prop in AppTypography. It are redundant styles



> We can use variant prop in AppTypography. It are redundant styles

removed